website/docs/r/compute_attached_disk.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: "google"
+page_title: "Google: google_compute_attached_disk"
+sidebar_current: "docs-google-compute-attached-disk"
+description: |-
+  Resource that allows attaching existing persistent disks to compute instances.
+---
+
+# google\_compute\_attached\_disk
+
+Persistent disks can be attached to a compute instance using [the `attach_disk`
+section within the compute instance configuration](https://www.terraform.io/docs/providers/google/r/compute_instance.html#attached_disk).
+However there may be situations where managing the attached disks via the compute
+instance config isn't preferable or possible. For example: attaching dynamic
+numbers of disks using the `count` variable.
+
+
+To get more information about attaching disks, see:
+
+* [API documentation](https://cloud.google.com/compute/docs/reference/rest/v1/instances/attachDisk)
+* [Resource: google_compute_disk](https://www.terraform.io/docs/providers/google/r/compute_disk.html)
+* How-to Guides
+    * [Adding a persistent disk](https://cloud.google.com/compute/docs/disks/add-persistent-disk)
+
+
+## Example Usage
+```hcl
+resource "google_compute_attached_disk" "default" {
+  disk = "${google_compute_disk.default.self_link}"
+  instance = "${google_compute_instance.default.self_link}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `instance` -
+  (Required)
+  `name` or `self_link` of the compute instance that the disk will be attached to.
+  If the `self_link` is provided then `zone` and `project` are extracted from the
+  self link. If only the name is used then `zone` and `project` must be defined
+  at a global level.
+
+* `disk` -
+  (Required)
+  `name` or `self_link` of the disk that will be attached.
+
+
+- - -
+
+* `project` -
+  (Optional)
+  The project that the referenced compute instance is a part of. If `instance` is referenced by it's
+  `self_link` the project defined in the link will take precedence.
+
+* `zone` -
+  (Optional)
+  The zone that the referenced compute instance is located within. If `instance` is referenced by it's
+  `self_link` the zone defined in the link will take precedence.
+
+* `auto_delete` -
+  (Optional)
+  Specifies whether the disk will be auto-deleted when the
+	instance is deleted (but not when the disk is detached from the
+	instance).
+
+* `device_name` -
+  (Optional)
+  Specifies a unique device name of your choice that is
+	reflected into the /dev/disk/by-id/google-* tree of a Linux operating
+	system running within the instance. This name can be used to
+	reference the device for mounting, resizing, and so on, from within
+	the instance.
+
+	If not specified, the server chooses a default device name to apply
+	to this disk, in the form persistent-disks-x, where x is a number
+	assigned by Google Compute Engine. This field is only applicable for
```

Comment:
you can remove the last sentence here since we're only dealing with persistent disks